packages/manager/src/queries/databases/databases.ts

```diff
@@ -53,10 +53,10 @@ export const databaseQueries = createQueryKeys('databases', {
   }),
   databases: {
     contextQueries: {
-      all: {
-        queryFn: getAllDatabases,
-        queryKey: null,
-      },
+      all: (filter?: Filter) => ({
```

Comment:
Can we default to an empty object
```suggestion
      all: (filter: Filter = {}) => ({
```